### PR TITLE
feat(kit): customize display control for input-files component

### LINKED ISSUE #2734: 🚀 - Allow more control over TuiFilesComponent display
### Which @taiga-ui/* package(s) are relevant/releated to the feature request?

kit

### Description

It's tricky customize the content of the files component, since it assumes there are 2 content elements (label and link). In the case where dragging occurs the only option for content is via the `texts` property.

I have the base display like so:

image

I would like to keep the same display on drag and change the background colour. However I can't do that because it computes the label while dragging as:

```
if (fileDragged) {
    return this.inputFileTexts$.pipe(
        map(texts => (multiple ? texts.dropMultiple : texts.drop)),
    );
}
```

### PR DESCRIPTION
**Pull request by oknimot**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [x] Documentation content changes

## What is the current behavior?

It's tricky to customize the content of the files component since it assumes there are 2 content elements (`label` and `link`). In the case where dragging occurs the only option for content is via the texts property.

Closes #2734 

## What is the new behavior?

If you want to override the content of the files component you are able to do this via the `content: PolymorpheusContent` input property. If you don't provide a `content` property value, the default behavior is the same and there will be displayed the default `link` and `label` elements.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Comment by splincode** [comment]
@oknimot 
image

**Comment by oknimot** [comment]
> @oknimot image

@splincode, thanks for noticing 🙌 ! Fixed in da0205ff886d1a267095319bf5a828790395bbd8

File: projects/demo/src/modules/components/input-files/input-files.template.html [hunk 1]
```diff
@@ -57,8 +57,7 @@
             <tui-input-files
                 [formControl]="control"
                 [accept]="accept"
-                [link]="link"
-                [label]="label"
+                [content]="content"
```
Comment (splincode) [inline projects/demo/src/modules/components/input-files/input-files.template.html:60]: add schematics, please

Comment (oknimot) [reply projects/demo/src/modules/components/input-files/input-files.template.html:60]: Fixed, @splincode thanks for your checks!

File: projects/cdk/schematics/ng-update/steps/migrate-input-files.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import {
```
Comment (splincode) [inline projects/cdk/schematics/ng-update/steps/migrate-input-files.ts:1]: try add example to `https://github.com/Tinkoff/taiga-ui/blob/main/projects/cdk/schematics/ng-update/tests/schematic-replace-html.spec.ts`

Comment (oknimot) [reply projects/cdk/schematics/ng-update/steps/migrate-input-files.ts:1]: Committed!

Comment (oknimot) [reply projects/cdk/schematics/ng-update/steps/migrate-input-files.ts:1]: I can see, that there is a conflict with the `schematic-replace-html.spec.ts` file. Can I resolve it?

Comment (splincode) [reply projects/cdk/schematics/ng-update/steps/migrate-input-files.ts:1]: rebase with main branch

**Comment by splincode** [comment]
not sure if you did the right rebase

**Comment by oknimot** [comment]
> not sure if you did the right rebase

Yup, you're right. Sorry 🫣, my fault. Going to fix it...

**Pull request closed without merging**